MekHQ/src/mekhq/campaign/rating/CamOpsReputation/ReputationController.java

```diff
@@ -623,8 +730,7 @@ public ReputationController generateInstanceFromXML(final Node workingNode) {
                 if (workingNode2.getNodeName().equalsIgnoreCase("averageSkillLevel")) {
                     this.averageSkillLevel = SkillLevel.valueOf(workingNode2.getTextContent().toUpperCase());
                 } else if (workingNode2.getNodeName().equalsIgnoreCase("averageExperienceRating")) {
-                    this.averageExperienceRating = Integer.parseInt(
-                        workingNode2.getTextContent().replaceAll("-", "_"));
+                    this.averageExperienceRating = Integer.parseInt(workingNode2.getTextContent().replaceAll("-", "_"));
                 } else if (workingNode2.getNodeName().equalsIgnoreCase("atbModifier")) {
                     this.atbModifier = Integer.parseInt(workingNode2.getTextContent());
```

Comment:
MathUtility parsing.... (I know this was EC doing it's thing but if the files going to change anyways, might a well fix a few things while at it)